internal/app/dogma/get_cmd.go

```diff
@@ -49,11 +49,18 @@ func (gf *getFileCommand) execute(c *cli.Context) error {
 	defer fd.Close()
 
 	if entry.Type == centraldogma.JSON {
-		b, err := marshalIndent(entry.Content)
-		if err != nil {
+		var b []byte
+		if isJSON(entry.Content) {
+			b, err = marshalIndent(entry.Content)
```

Comment:
I think this workaround is good but it has a lot of duplicated code.
How about moving the logic into [`marshalIndent()`](https://github.com/ikhoon/centraldogma-go/blob/52ed8f59381263a82adcde3fa11b7cbadfa097e0/internal/app/dogma/util.go#L280) method?
```go
func marshalIndent(src []byte) ([]byte, error) {
	if isJSON(src) { ... }
	dst := new(bytes.Buffer)
	err := json.Indent(dst, src, "", "  ")
	return dst.Bytes(), err
}
```

Reply:
Yeah, that a good idea. But I have to change the name `marshalIndent` to something else. Any suggestion for that?

Reply:
Do you mean that `marsharIndent` returns `nil` when the content is a string? so the code should be
```go
b, err = marshalIndent(entry.Content)
if err != nil {
  return err
}
if b == nil {
  b = entry.Content
}
```

Reply:
> But I have to change the name marshalIndent to something else. Any suggestion for that?
How about adding `safeMarshalIndent()`? 😀
```go
func safeMarshalIndent(src []byte) ([]byte, error) {
	if isJSON(src) {
		return marshalIndent(src)
	} else {
		return src, nil
	}
}
```

Reply:
I like it. Thanks!